CsvParser/csvParser.go

```diff
@@ -78,18 +87,18 @@ func CsvParser(filePath string) [100]incident.DefaultIncident {
 		defaultIncident.Init(goodTimeStart, goodTimeEnd, timeStart, timeEnd, avgGoodDS,
 			avgBadDS, severity, testsAffected)
 
-		incidentArray[i] = *defaultIncident
+		defaultIncidents = append(defaultIncidents, *defaultIncident)
+		i = i + 1
 
 	}
-
-	return incidentArray
+	return defaultIncidents
 }
 
 //* This function takes in an array of 100 default incidents because that is what is provided by the csvParser above *//
-func convertDefaultIncidentToIncident(arr [100]incident.DefaultIncident) []incident.Incident {
-	incidentArr := make([]incident.Incident, len(arr), len(arr))
-	for i := range arr {
-		incidentArr[i] = &arr[i]
+func convertDefaultIncidentToIncident(defaultIncidents []incident.DefaultIncident) []incident.Incident {
```

Comment:
where is this function gonna get used? Sometime in the future potentially. I wrote another one in my branch that takes one default incident and convert it into an IncidentData, which is a dummy class we have been using to dump things in a .json file without compromising security (private data members)

Reply:
I don't know... But the only thing I changed was the data structure